study.md

```diff
@@ -54,5 +54,5 @@ like enumerables?  How are they different?  Please compare and contrast these
 types.
 
 ```md
-<!-- your answer here -->
+Streams are like enumerables because they are processed one item at a time, like down a list.  They are different because streams are a potentially infinite source of data, whereas an enumerable is finite.  
```

Comment:
Streams should generally be treated as infinite, even though they can be finite, and enumerables should generally be treated as finite, even though they can be infinite.  An example of an infinite enumerable is one that  returns a digit of π on each iteration.